enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/service/CommonOtpService.java

```diff
@@ -121,7 +129,11 @@ public OtpVerifyResponse verifyOtpCode(String processId, OwnerId ownerId, String
         response.setOnboardingStatus(process.getStatus());
         response.setExpired(expired);
         response.setVerified(verified);
-        response.setRemainingAttempts(maxFailedAttempts - failedAttempts);
+        if (otp.getStatus() == OtpStatus.ACTIVE) {
```

Comment:
Nit: Or could be extracted to a method and use `response.setRemainingAttempts(calculateRemainingAttempts(otp))`

Reply:
Sure, why not.